functionary/ui/static/css/functionary.css

```diff
@@ -52,4 +52,23 @@ tr.htmx-swapping td {
 }
 body div aside {
     max-width: fit-content;
-}
\ No newline at end of file
+}
+
+/* table sorting indicators */
+th.asc::after {
+    content: '\f077';    /* chevron-up */
+    font-family: var(--fa-style-family,"Font Awesome 6 Free");
+    color: var(--bs-primary);
+    font-size: 0.75em;
+    line-height: .05em;
+    vertical-align: -.1em;
+    padding-left: .5em;
```

Comment:
My only suggestion would be to just consolidate the similar styles into one class, like:
```css
th.asc::after,
th.desc::after {
    font-family: var(--fa-style-family,"Font Awesome 6 Free");
    color: var(--bs-primary);
    font-size: 0.75em;
    line-height: .05em;
    vertical-align: -.1em;
    padding-left: .5em;
}
th.asc::after {
    content: '\f077';    /* chevron-up */
}
th.desc::after {
    content: '\f078';    /* chevron-down */
}
```

Also, for the `vertical-align`, is there any reason to not just let the element be vertically aligned with the middle of the element, rather than `-.1em`?

Reply:
Yeah, good call on consolidating these.

The `vertical-align` was an artifact from a some alignment wonkiness when I was using a different icon.  I removed it from the `desc` but forgot to take it out of the `asc` apparently.  It's gone now.